packages/tsconfig/base.json

```diff
@@ -24,8 +27,9 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "resolveJsonModule": true,
-    "target": "ES2020",
-    "module": "ES2022"
+    "target": "esnext",
```

Comment:
This feels too current. Think ES2022 covers like 90% of our users. We're not sure esnext won't release a new feature that could be too new.